app-drawer/app-drawer.html

```diff
@@ -137,7 +139,8 @@
       transition: opacity 0.2s ease;
 
       -webkit-transform: translateZ(0);
-      transform:  translateZ(0);
+      transform: translateZ(0);
+      will-change: opacity;
```

Comment:
But the idea of adding `will-change` is that you **don't** need to do the `transform: translateZ(0);` hack.  And hence my suggestion to remove it.